Sources/TokamakDOM/Views/Gestures/GestureView.swift

```diff
@@ -0,0 +1,41 @@
+// Copyright 2020 Tokamak contributors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+//  Created by Szymon on 16/7/2023.
+//
+
+import JavaScriptKit
+import TokamakCore
+import TokamakStaticHTML
+import Foundation
+
+extension TokamakCore.GestureView: DOMPrimitive {
+    var renderedBody: AnyView {
+        switch G.Body.self {
+        case is TapGesture.Type:
+            return AnyView(_TapGestureView(gesture: $gesture, content: content))
+        case is LongPressGesture.Type:
+            return AnyView(_LongPressGestureView(gesture: $gesture, content: content))
+        case is DragGesture.Type:
+            return AnyView(_DragGestureView(gesture: $gesture, content: content))
+        default:
+            return AnyView(
```

Comment:
Does this path need to be updated to support recursive custom gestures?

Reply:
structure changed, outdated.